lib/resty/openssl/include/x509_vfy.lua

```diff
@@ -80,10 +85,23 @@ elseif OPENSSL_11_OR_LATER then
   ffi.cdef [[
     // STACK_OF(X509)
     OPENSSL_STACK *X509_STORE_CTX_get0_chain(X509_STORE_CTX *ctx);
+    typedef int (*X509_STORE_CTX_check_revocation_fn)(X509_STORE_CTX *ctx);
+    // STACK_OF(X509)
+    void X509_STORE_CTX_set0_verified_chain(X509_STORE_CTX *ctx, OPENSSL_STACK *sk);
   ]];
   _M.X509_STORE_CTX_get0_chain = C.X509_STORE_CTX_get0_chain
 end
 
+if OPENSSL_3X then
+  ffi.cdef [[
+    X509_STORE_CTX_check_revocation_fn X509_STORE_CTX_get_check_revocation(const X509_STORE_CTX *ctx);
```

Comment:
we actually don't need to distinguish `const` in ffi cdef, it's only an decorator for C compiler.

Reply:
Applied